src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,30 @@
+@import "./utils/variables";
+
+.catalog {
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+  grid-template-columns: repeat(4, $card-width);
+
+  @media (max-width: 1023px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (max-width: 767px) {
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (max-width: 487px) {
+    grid-template-columns: repeat(1, $card-width);
+  }
```

Comment:
Your cards start to placing in 2 columns from 487px, but not 488px, as was mentioned in requirements.

https://prnt.sc/MNU-YSRq3BG9

Reply:
I know that. I've googled that problem. This is the issue with browser rounding and if I make 488px it swipes to 489px so I can't set perfect match. Maybe you know how to fix that?